src/components/Cabinet/Cabinet.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
Please, lowercase the name of the file and the folder

Reply:
Renamed